infra/blueprint-test/pkg/gcloud/gcloud.go

```diff
@@ -107,6 +117,18 @@ func Run(t testing.TB, cmd string, opts ...cmdOption) gjson.Result {
 	return gjson.Parse(op)
 }
 
+// TerraformVet executes gcloud beta terraform vet
+func TerraformVet(t testing.TB, planFilePath string, policyLibraryPath string) gjson.Result {
+	op, err := RunCmdE(t, stringFromTextAndArgs(append([]interface{}{"beta terraform vet %s --policy-library=%s"}, planFilePath, policyLibraryPath)...))
```

Comment:
Can't we just use Sprintf rather than stringFromTextAndArgs  since we control the format strings?

Reply:
fixed